### PR TITLE
Add examples tests

### LINKED ISSUE #157: Automated Tests
Define and implement some basic unit or integration tests to be run as part of the CI.

### PR DESCRIPTION
**Pull request by luniki**
I added some very basic tests just to prove the concept. Unfortunately the original opencast-studio API code (for uploading and connection testing) is not easily mockable. This has to be refactored before integrations tests are possible.

Nevertheless this PR demonstrates how to write and employ unit/integration tests with opencast-studio.

Closes #157

### CODE REVIEW

File: src/ui/notification.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ const Notification = props => (
   <div
     sx={{
       ':not(:last-child)': { marginBottom: '1.5rem' },
-      backgroundColor: props.isDanger ? '#ff3860' : 'whitesmoke',
-      color: props.isDanger ? '#fff' : 'currentColor',
+      backgroundColor: props.isdanger ? '#ff3860' : 'whitesmoke',
+      color: props.isdanger ? '#fff' : 'currentColor',
       borderRadius: 2,
       padding: 3,
       position: 'relative'
```
Comment (LukasKalbertodt) [inline src/ui/notification.js]: Regarding the renaming of `isDanger` to `isdanger`: the problem is that when you add all the props as attributes of this `div` below (`{...props}`), the `isDanger` property (which is not a valid HTML attribute) is added as well. By renaming it to `isdanger`, the warning is just silenced and we emit this invalid HTML attribute. The proper solution is to exclude `isDanger` from the properties that are added as HTML attributes. This works:

```js
const Notification = props => {
  const { isDanger, ...rest } = props;
  return (<div
    sx={{
      ':not(:last-child)': { marginBottom: '1.5rem' },
      backgroundColor: isDanger ? '#ff3860' : 'whitesmoke',
      color: isDanger ? '#fff' : 'currentColor',
      borderRadius: 2,
      padding: 3,
      position: 'relative'
    }}
    {...rest}
  />)
};
```

See [this StackOverflow Q&A](https://stackoverflow.com/questions/49358560/react-wrapper-react-does-not-recognize-the-staticcontext-prop-on-a-dom-elemen) for more information.

I think this should be changed (and of course, the renaming in the three other files should be reverted).

Comment (luniki) [reply src/ui/notification.js]: Thank you for researching this.

File: src/ui/notification.js [hunk 2]
```diff
@@ -2,17 +2,17 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui';
 
-const Notification = props => (
+const Notification = ({isDanger, ...rest}) => (
```
Comment (LukasKalbertodt) [inline src/ui/notification.js]: Ah cool. I didn't know one can already "take apart" an object in the argument list. That's handy.

Comment (luniki) [reply src/ui/notification.js]: Yeah, that's really nice. Learned this from Dr Axel:

https://exploringjs.com/impatient-js/ch_destructuring.html#where-can-we-destructure

**Pull request merged**